Rakefile

```diff
@@ -208,10 +212,15 @@ task :release, :tag do |t, args|
     config.key = api_token
   end if api_token
 
+  require_relative "rakelib/link_transformer.rb"
```

Comment:
Style: you can omit the `.rb` when specifying a Ruby file in a require statement.